<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gpiotest</class>
 <widget class="QMainWindow" name="gpiotest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gpiotest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>721</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#groupBox{

	font-size : 20px;
	border-color : rgb(255,255,255);
	border : 3px solid;
}</string>
    </property>
    <property name="title">
     <string>PIN SELECT</string>
    </property>
    <widget class="QComboBox" name="Port_box">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Agency FB&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Pin_box">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_BT">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SET</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#groupBox_2{


	font-size : 20px;
	border-color : white;
	border : 3px solid;



}</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="Output_BT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OUTPUT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Input_BT">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>INPUT</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
